v2/pkg/api/v1alpha2/types_config.go

```diff
@@ -47,6 +65,26 @@ type Mirror struct {
 	Samples []SampleImages `json:"samples,omitempty"`
 }
 
+// Delete defines the configuration for content types within the imageset.
+type Delete struct {
+	// Platform defines the configuration for OpenShift and OKD platform types.
+	Platform Platform `json:"platform,omitempty"`
+	// Operators defines the configuration for Operator content types.
+	Operators []Operator `json:"operators,omitempty"`
+	// AdditionalImages defines the configuration for a list
+	// of individual image content types.
+	AdditionalImages []Image `json:"additionalImages,omitempty"`
+	// Helm define the configuration for Helm content types.
+	Helm Helm `json:"helm,omitempty"`
+	// BlockedImages define a list of images that will be blocked
+	// from the mirroring process if they exist in other content
+	// types in the configuration.
+	BlockedImages []Image `json:"blockedImages,omitempty"`
```

Comment:
Maybe let's omit this for now: I don't know how blockedImages would translate for a delete feature. WDYT?

Reply:
Yes agreed not used